src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ResetPassword.cshtml

```diff
@@ -24,20 +26,20 @@
 
                 <div class="form-group">
                     <div class="text_field text_field_default">
-                        <input asp-for="Password" placeholder="Password" />
+                        <input asp-for="Password" placeholder=@Password />
                     </div>
                     <span asp-validation-for="Password" class="text-danger"></span>
                 </div>
 
                 <div class="form-group">
                     <div class="text_field text_field_default">
-                        <input asp-for="ConfirmPassword" placeholder="Potwierdź hasło" />
+                        <input asp-for="ConfirmPassword" placeholder=@ConfirmPassword />
                     </div>
                     <span asp-validation-for="ConfirmPassword" class="text-danger"></span>
                 </div>
 
                 <div class="text-center mb-30">
-                    <button type="submit" class="chris_btn">Reset</button>
+                    <button type="submit" class="chris_btn">@ResetShort</button>
```

Comment:
why can't you use `Reset`?

Reply:
Right now we have:
Reset -> Resetuj
ResetShort -> Reset

Reply:
I can change Resetuj to Reset and make only one translation 'Reset' -> 'Reset'

Reply:
done